ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/download/_cq_dialog/.content.xml

```diff
@@ -105,6 +105,31 @@
                                             fieldLabel="ZIP File Name"
                                             name="./zipFileName"
                                             required="{Boolean}true"/>
+                                    <heading
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/heading"
+                                        level="{Long}4"
+                                        text="Maximum Content Size for Downloads"/>
+                                    <well
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/well">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <max-content-size
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                                fieldDescription="Sets a maximum size limit for downloads (in bytes). Recommended to match size in AssetDownloadServlet OSGi config. Leave blank for no maximum content size."
```

Comment:
If it's left blank, the download will fail if the AssetDownloadServlet OSGi configuration is exceeded right? The description makes it sound like if you leave it blank, there is no limit.

Reply:
yeah agreed, unless you are on 6.3/6.4 and don't have the updated AssetDownloadServlet. Can update the description to make it more clear, that this would leave off any UI checks before the download